dagshub/data_engine/model/query_result.py

```diff
@@ -622,11 +697,44 @@ def visualize(self, visualizer: Literal["dagshub", "fiftyone"] = "dagshub", **kw
 
             return sess
 
+    def annotate_with_mlflow_model(
+        self,
```

Comment:
Why does this function exist? Is it going to be doing something different later?
If they are doing the same thing basically, I think it will only lead to confusion as to which one needs to be used.

Also the docstring is wrong, don't see any Label Studio happening here

Reply:
I plan to add an additional parameter (the "Coming soon!" one), where instead of performing annotation locally it will create a remote LS model backend and perform inference using that.

Nonetheless, I think it's a good idea to have it because this use case might go under the radar, despite it as it currently stands just being a function wrapper.

Reply:
Thank you for the docstring catch, I've been slipping up with those. I'll fix that now!